build.rs

```diff
@@ -0,0 +1,21 @@
+use std::process::Command;
+
+// Example custom build script.
+fn main() -> std::io::Result<()> {
+    
+    println!("cargo:warning=QBE source has changed. Recompiling...");
+
+    let success = Command::new("make")
```

Comment:
This will fail on BSDs as their default make is not GNU, maybe check for `gmake` and then fall-back to normal make?